crates/router/src/connector/adyen.rs

```diff
@@ -72,6 +73,29 @@ impl ConnectorCommon for Adyen {
     }
 }
 
+impl ConnectorValidation for Adyen {
+    fn validate_capture_method(
+        &self,
+        capture_method: storage_enums::CaptureMethod,
+    ) -> CustomResult<(), errors::ConnectorError> {
+        let unsupported_capture_method = match capture_method {
+            enums::CaptureMethod::Automatic | enums::CaptureMethod::Manual => None,
+            enums::CaptureMethod::ManualMultiple => Some("manual_multiple"),
+            enums::CaptureMethod::Scheduled => Some("schedule"),
+        };
+        if let Some(capture_method) = unsupported_capture_method {
+            Err(errors::ConnectorError::NotImplemented(format!(
+                "{} for {}",
+                capture_method,
+                self.id()
+            ))
+            .into())
+        } else {
+            Ok(())
+        }
```

Comment:
```suggestion
match capture_method {
            enums::CaptureMethod::Automatic |
            enums::CaptureMethod::Manual => Ok(()),
            enums::CaptureMethod::ManualMultiple |
            enums::CaptureMethod::Scheduled => Err(errors::ConnectorError::NotImplemented(format!(
                "{} for {}",
                capture_method,
                self.id()
            ))
            .into()),
        }
```

Try to use single match and return the respective value